DESCRIPTION

```diff
@@ -35,6 +36,7 @@ Imports:
     ids,
     jsonlite,
     magrittr,
+    promises,
```

Comment:
I think we should mostly avoid adding new dependencies. For icons we already have fontawesome (which we would already have because of shiny) and for promises I'll make a comment on the ExtendedTask part

Reply:
`promises` is already a dependency for shiny, but I'll see if I can get around bsicons.